pkg/utils/config.go

```diff
@@ -0,0 +1,164 @@
+package utils
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/satori/go.uuid"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+	"os"
+	"os/user"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"text/template"
+)
+
+var defaultEnvConfig = `#generated by eden
+adam-dist: {{ .Dist }}/adam
+image-dist: {{ .Dist }}/images
+downloader-dist: {{ .Dist }}/eve/dist/{{ .Arch }}
+downloader-save: true
+eserver-port: 8888
+eserver-pid: {{ .Dist }}/eserver.pid
+eserver-log: {{ .Dist }}/eserver.log
+eve-arch: {{ .Arch }}
+eve-os: {{ .OS }}
+eve-accel: true
+adam-port: 3333
+eve-serial: 31415926
+eve-cert: {{ .Dist }}/certs/onboard.cert.pem
+eve-config: {{ .Dist }}/qemu.conf
+eve-pid: {{ .Dist }}/eve.pid
+eve-log: {{ .Dist }}/eve.log
+adam-force: true
+certs-dist: {{ .Dist }}/certs
+domain: mydomain.adam
+ip: {{ .IP }}
+uuid: {{ .UUID }}
+hv: kvm
+eve-firmware: {{ .Dist }}/eve/dist/amd64/OVMF.fd
```

Comment:
This is actually great as a standalone config item, but can you please make sure that if I set it to empty value than the default firmware (SeaBIOS) is being used by qemu?

Also, can you please make sure that it would be possible to specify a pair of files as a value here. E.g.
```
eve-firmware: foo.bin, bar.vars
```
So that we can use the pflash style of passing UEFI firmware to qemu. E.g.
```
-drive if=pflash,format=raw,unit=0,readonly,file=foo.bin -drive if=pflash,format=raw,unit=1,file=bar.vars
```

Reply:
Done:
https://github.com/lf-edge/eden/blob/ea676f293e39ed98b8fd87103d1a4880316554e5/pkg/utils/qemu.go#L20-L38